src/Symfony/Component/Notifier/Bridge/Telnyx/TelnyxTransportFactory.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\Telnyx;
+
+use Symfony\Component\Notifier\Exception\UnsupportedSchemeException;
+use Symfony\Component\Notifier\Transport\AbstractTransportFactory;
+use Symfony\Component\Notifier\Transport\Dsn;
+use Symfony\Component\Notifier\Transport\TransportInterface;
+
+/**
+ * @author Vasilij Duško <vasilij@prado.lt>
+ */
+final class TelnyxTransportFactory extends AbstractTransportFactory
+{
+    /**
+     * @return TelnyxTransport
+     */
+    public function create(Dsn $dsn): TransportInterface
+    {
+        $scheme = $dsn->getScheme();
+
+        if ('telnyx' !== $scheme) {
+            throw new UnsupportedSchemeException($dsn, 'telnyx', $this->getSupportedSchemes());
+        }
+
+        $apiKey = $this->getUser($dsn);
+        $from = $dsn->getRequiredOption('from');
+        $profileId = $dsn->getOption('profileId', '');
```

Comment:
```suggestion
        $profileId = $dsn->getOption('profileId');
```

and make the property and argument nullable in the transport

Reply:
Good point